src/config/groups/keys.cpp

```diff
@@ -59,8 +59,6 @@ Keys::Keys(
         auto key_list = group_keys();
         members.replace_keys(key_list, /*dirty=*/false);
         info.replace_keys(key_list, /*dirty=*/false);
-    } else if (admin()) {
-        rekey(info, members);
```

Comment:
The comment for the `rekey()` function in the header needs an update with this change about when it should be called, something along the lines of "This must also be called when creating a new group from scratch (but not when merely creating a new group *object* from new incoming group info such as from config data pushed by another device)."